docs/development/install.rst

```diff
@@ -184,3 +184,38 @@ For building this documentation,
 see :doc:`docs`.
 
 And for setting up for the front end development, see :doc:`standards`.
+
+Troubleshooting
+---------------
+
+You might face issues while installing.
+
+Error: `SocialApp matching query does not exist.`
+
+Please follow the guidelines for installation given here:
+
+- Edit `readthedocs/settings/base.py`
+  Add entry to the authentication backend section:
+  `'allauth.account.auth_backends.AuthenticationBackend'`
```

Comment:
I think this is already added to the base settings

Reply:
Are you sure?

Reply:
Yes :)
https://github.com/readthedocs/readthedocs.org/blob/4b38e8228f2df43cd81823139f525d999f6e4467/readthedocs/settings/base.py#L176-L181

Reply:
Cool.